software/controller/integration_tests/eeprom_test.h

```diff
@@ -82,14 +87,19 @@ static Debug::Command::VarHandler var_command;
 static Debug::Command::TraceHandler trace_command(&trace);
 static Debug::Command::EepromHandler eeprom_command(&eeprom);
 
-static Debug::Interface debug(&debug_uart, &trace, 12, Debug::Command::Code::Mode, &mode_command,
-                              Debug::Command::Code::Peek, &peek_command, Debug::Command::Code::Poke,
-                              &poke_command, Debug::Command::Code::Variable, &var_command,
-                              Debug::Command::Code::Trace, &trace_command,
-                              Debug::Command::Code::EepromAccess, &eeprom_command);
+static Debug::Interface debug(&debug_uart);
 
 void RunTest() {
   hal.Init();
+
+  /// \TODO are all the handlers really necessary for this integration test?
+  debug.add_handler(Debug::Command::Code::Mode, &mode_command);
+  debug.add_handler(Debug::Command::Code::Peek, &peek_command);
+  debug.add_handler(Debug::Command::Code::Poke, &poke_command);
+  debug.add_handler(Debug::Command::Code::Variable, &var_command);
+  debug.add_handler(Debug::Command::Code::Trace, &trace_command);
+  debug.add_handler(Debug::Command::Code::EepromAccess, &eeprom_command);
+
```

Comment:
Maybe I won't touch these for now, because I see that this integration test could use even more work. Let's leave the TODO and take care of it in another PR.